src/test/resources/com/tresys/cobol1/tests.tdml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tdml:testSuite 
+ xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+ xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" 
+ xmlns:xs="http://www.w3.org/2001/XMLSchema"
+ xmlns:ex="http://example.com" 
+ xmlns:daf="urn:ogf:dfdl:2013:imp:daffodil.apache.org:2018:ext"
+ defaultConfig="cfg"
+ defaultRoundTrip="onePass">
+ 
+  <tdml:defineConfig name="cfg">
+    <daf:tunables>
+      <daf:suppressSchemaDefinitionWarnings>
+        deprecatedPropertySeparatorPolicy
+        encodingErrorPolicyError
+        emptyElementParsePolicyError
+      </daf:suppressSchemaDefinitionWarnings>
+    </daf:tunables>
+  </tdml:defineConfig>
+
+  <tdml:unparserTestCase name="test1" model="com/tresys/cobol1/xsd/cobol1.dfdl.xsd">
+    <tdml:document>
+      <tdml:documentPart type="text" encoding="ebcdic-cp-us">1234567890123456789</tdml:documentPart>
+      <tdml:documentPart type="byte">0020161C</tdml:documentPart>
+      <tdml:documentPart type="byte">0020161C</tdml:documentPart>
+      <tdml:documentPart type="byte">0020161C</tdml:documentPart>
+      <tdml:documentPart type="byte">0020161C</tdml:documentPart>
+      <tdml:documentPart type="byte">99999C</tdml:documentPart>
+      <tdml:documentPart type="byte">270f</tdml:documentPart>
```

Comment:
This binary number may have an alignment to a 2-byte boundary - depends on the cobol implementation. Should set this up for typical IBM s390 Cobol. 

The same is true for the other binary numbers in the data (could be 2, 4, or 8 byte boundaries).